src/main/java/com/google/sps/servlets/KeywordServlet.java

```diff
@@ -39,10 +71,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   /**
    * Generates and returns a JSON usable by Mustache.
    */
-  private String jsonBuilder(Collection<String> toJson) {
+  private String jsonBuilder(Map<Entity, Float> toJson) {
```

Comment:
[optional] Would it be better to make a new list for the json than clearing the old terms list?

Reply:
I'm not quite understanding what you mean. Would you be able to clarify?